nectar/downloaders/threaded.py

```diff
@@ -30,8 +18,8 @@
 _logger = getLogger(__name__)
 
 DEFAULT_MAX_CONCURRENT = 5
-DEFAULT_BUFFER_SIZE = 8192 # bytes
-DEFAULT_PROGRESS_INTERVAL = 5 # seconds
+DEFAULT_BUFFER_SIZE = 8192  # bytes
+DEFAULT_PROGRESS_INTERVAL = 5  # seconds
 DEFAULT_RETRIES = 3
```

Comment:
I would set this to 1. If we try a request, then re-try, and it still fails, the chance of it working on the third or fourth request is small. Continuing to hammer a struggling remote server is more likely to make its problems worse than to help anything.